### ISSUE TITLE
Albino lullaby won't show up in headset just on monitor

### ISSUE THREAD

**Issue by pearce31** [LibreVR/Revive#296]
I tried the new preview and the release before that. I think i even tried the release before that one. The installer and injector. I bought it from oculus home. I'm using 64 bit win 10. Thanks.

**Comment by nomansfry** [comment]
I am sorry, maybe I am missing something here. But why would you need to buy Albino Lullaby off of Oculus home? I thought it already had Vive support.

**Comment by pearce31** [comment]
It was only 3 dollars for the Halloween sale

**Comment by LibreVR** [comment]
Since it already has Vive support I don't think I'll buy it just to try and get it working through Revive. Perhaps it will be fixed later, but for a game with native Vive support there's not much incentive.

@pearce31 Your best bet is to contact the developer and ask for a Steam key.

**Comment by pearce31** [comment]
Yeah figures... thanks nomansfry for sticking your nose where it don't belong! BTW I'm not using a vive. Thanks a lot for the help really appreciate it!

**Comment by LibreVR** [comment]
You're probably using VRidge, that should also work with the Steam version. Like I said, try asking the devs for a Steam key.

**Comment by pearce31** [comment]
OK I asked them. Fingers crossed

**Comment by pearce31** [comment]
OK the awesome owner of the game gave me a key

**Comment by LibreVR** [comment]
Good to hear it, enjoy. I'll keep this issue open so the Oculus Home version can be fixed at some point.

**Comment by pearce31** [comment]
Alrighty, thanks
